test/test_torch.py

```diff
@@ -6987,80 +6988,74 @@ def test_logical_and(self, device):
     def test_logical_or(self, device):
         self._test_logical(device, 'logical_or', [10, 0, 1, 0], [1, 0, 0, 10], [1, 0, 1, 1])
 
-    def test_clamp(self, device):
-        m1 = torch.rand(100, device=device).mul(5).add(-2.5)  # uniform in [-2.5, 2.5]
-        # just in case we're extremely lucky.
-        min_val = -1
-        max_val = 1
-        m1[1] = min_val
-        m1[2] = max_val
+    def generate_clamp_baseline(self, device, min_vals=None, max_vals=None, with_nans=False):
```

Comment:
Please add a comment to describing what this function does and remove the default values for clarity but make them kwarg-only: `(self, device, * min_vals, max_vals, with_nans)`